lab/lab1/part3-data-transformation.html

```diff
@@ -63,49 +63,109 @@
 
     ===================== */
 
-    var data = bikeArrayClean;
+    var map = L.map('map', {
+      center: [39.9522, -75.1639],
+      zoom: 14
+    });
+
+    var Stamen_TonerLite = L.tileLayer('http://stamen-tiles-{s}.a.ssl.fastly.net/toner-lite/{z}/{x}/{y}.{ext}', {
+      attribution: 'Map tiles by <a href="http://stamen.com">Stamen Design</a>, <a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a> &mdash; Map data &copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>',
+      subdomains: 'abcd',
+      minZoom: 0,
+      maxZoom: 20,
+      ext: 'png'
+    }).addTo(map);
 
     /* =====================
 
     Start code to filter data
 
     ===================== */
+    var data = bikeArrayDirty;
+
+    var Twenty = [];
+
+    for(let i=0; i<data.length; i++){
+      if(data[i][3]>20){
+        Twenty.push(data[i]);
+      }
+    }
+
+      console.log(Twenty);
 
+      /* =====================
+
+      End code to filter data
+
+      ===================== */
 
 
     /* =====================
 
-    End code to filter data
+    Start code to add markers to map
 
     ===================== */
+    var AddMarkers = (obj) => {
+      return L.marker([obj[1],obj[0]])
+              .bindPopup(obj[2]);
+    };
+
+    for (let i=0; i<Twenty.length;i++){
+      AddMarkers(Twenty[i]).addTo(map);
+    }
+    //data.map(addMarkers)
+
+    // for(let i=0; i<data.length; i++){
+    //   L.marker([data[i][1],data[i][0]])
+    //   .addTo(map)
+    //   .bindPopup(data[i][2]);
+    // }
 
-    var map = L.map('map', {
-      center: [39.9522, -75.1639],
-      zoom: 14
-    });
+    /* =====================
 
-    var Stamen_TonerLite = L.tileLayer('http://stamen-tiles-{s}.a.ssl.fastly.net/toner-lite/{z}/{x}/{y}.{ext}', {
-      attribution: 'Map tiles by <a href="http://stamen.com">Stamen Design</a>, <a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a> &mdash; Map data &copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>',
-      subdomains: 'abcd',
-      minZoom: 0,
-      maxZoom: 20,
-      ext: 'png'
-    }).addTo(map);
+    End code to add markers to map
+
+    ===================== */
 
     /* =====================
 
-    Start code to add markers to map
+    Start code to clean the dirty bike data and add markers
 
     ===================== */
 
+    var Dirty = bikeArrayDirty
```

Comment:
Capitalizing the `D` makes it fail later. In general in JS, it goes against convention to capitalize the first letter of a variable.